cmd/clusterctl/cmd/topology_plan.go

```diff
@@ -121,11 +122,11 @@ func runTopologyPlan() error {
 	for _, f := range tp.files {
 		raw, err := os.ReadFile(f) //nolint:gosec
 		if err != nil {
-			return errors.Wrapf(err, "failed to read input file %q", f)
+			return pkgerrors.Wrapf(err, "failed to read input file %q", f)
```

Comment:
While we're here, should we move these to be fmt.Errorf instead?

Reply:
Not sure I thought I'll use the pkg/errors library as that seems to be sort of the standard in CAPI :)

Reply:
Not sure if I got your question correctly.

I think as of today github.com/pkg/errors is still the standard in CAPI that we use everywhere to create and wrap errors. Are you asking to use the Go SDK stuff instead? If yes, I would defer that to a separate issue / PR / discussion around if we want to migrate away from pkg/errors.

Reply:
Merged this PR for now as we would like to use it to debug e2e tests. Happy to continue to conversation and follow-up accordingly

Reply:
The packages are compatibile with each other, especially the wrap/unwrap; we could just use those going forward

Reply:
Isn't there a difference in the amount of information they store in the errors? (pkg/errors stores the entire stacktrace which then gets logged if JSON logging is used)